calendar_v17/odoo_office365_calendar/models/calendar_event.py

```
@@ -0,0 +1,651 @@
+                            "Type": "numbered",
+                            "StartDate": start_date_str,
+                            "NumberOfOccurrences": rec.count
+                        }
+                    else:
+                        range_data = {
+                            "Type": "endDate",
+                            "StartDate": start_date_str,
+                            "EndDate": rec.until.strftime("%Y-%m-%d") if rec.until else None
+                        }
+
+                    recurrence_data = {
```

Ruff (0.11.9)

Local variable `recurrence_data` is assigned to but never used

Remove assignment to unused variable `recurrence_data`

(F841)